### DISCUSSION TITLE
Featherpad - file open/save dialog doesn't show mounts

### DISCUSSION THREAD

**Discussion by maplee0** [lxqt/lxqt#2451] [General]
Hello!
Sorry for the bother. I'm wondering.. is it possible to show mount points in Featherpad file dialogs? Like KDE does.
Debian 11, icewm.
What's needed? Dependency?

**Comment by tsujan** [comment]
The file dialog belongs to Qt and its plugins. not to FeatherPad.

If `Preferences → Window → Native file dialog` is checked (which is, by default), then the *native file dialog* will be used. The existence of a native file dialog depends on whether your DE provides the required Qt plugins (LXQt and KDE do). Native file dialogs are usually rich and show mounts, among other things (settings, thumbnails,...).

But if your DE doesn't have any Qt plugin, then the default Qt file dialog will be used. It's minimal and can't show many things.

Being a minimal DE, `icewm` doesn't have any Qt plugin.

The above explanation is true for all Qt apps, not just FeatherPad.

**Comment by tsujan** [comment]
BTW, FeatherPad doesn't belong to LXQt. The proper place for asking questions about it is https://github.com/tsujan/FeatherPad/discussions

**Comment by maplee0** [comment]
Many thanks!

**Reply by Butterfly** [reply]
A quickie from a user hoping to help speed along a fix via the recommended FeatherPad Developers route: I attempted this via `Open > Computer > File System`. Didn't work maybe due to permissions. That was odd.

Short term workaround: My FeatherPad is showing existing Bookmarks in left-hand tree. I make those via Thunar file manager (highlight a destination and click CTRL+D). It's nice to see that being interactive with FeatherPad. FeatherPad gets a hat tip for additionally prominently labeling those as bookmarks. A nice, user-friendly touch. Salute! :100:
